gearsyncd/gearsyncd.cpp

```diff
@@ -105,7 +105,7 @@ int main(int argc, char **argv)
         cerr << "Exception \"" << e.what() << "\" had been thrown in gearsyncd daemon" << endl;
         return EXIT_FAILURE;
     }
-    return 1;
+    return 0;
```

Comment:
You return `EXIT_FRAILURE` above. Suggest returning `EXIT_SUCCESS` (which is defined as 0) here for consistency.